Rectangle/TodoMode/TodoManager.swift

```diff
@@ -70,17 +76,28 @@ class TodoManager {
     
     private static func shiftWindowOffSidebar(_ w: AccessibilityElement, screenFrame: CGRect) {
         var rect = w.rectOfElement()
-
-        if (rect.maxX > (screenFrame.maxX - CGFloat(Defaults.todoSidebarWidth.value))) {
-            // Shift it to the left
-            rect.origin.x = max (0, rect.origin.x - (rect.maxX - (screenFrame.maxX - CGFloat(Defaults.todoSidebarWidth.value))))
-
-            // If it's still too wide, scale it down
-            if (rect.origin.x == 0) {
-                rect.size.width = min(rect.size.width, screenFrame.maxX - CGFloat(Defaults.todoSidebarWidth.value))
+        
+        if (Defaults.todoLeft.enabled) {
```

Comment:
I wonder if this chunk can be simplified a little so you're not repeating the same pattern of code twice (so it's easier to maintain going forward, etc.)

Perhaps define some variables that are dependent on whether `todoLeft` is enabled and then have one chunk of code that just sets values based on those vars